ci/test/04_install.sh

```diff
@@ -81,7 +99,7 @@ if [ -n "$PIP_PACKAGES" ]; then
     IN_GETOPT_BIN="/usr/local/opt/gnu-getopt/bin/getopt" ${CI_RETRY_EXE} pip3 install --user $PIP_PACKAGES
   else
     # shellcheck disable=SC2086
-    ${CI_RETRY_EXE} CI_EXEC pip3 install --user $PIP_PACKAGES
+    ${CI_RETRY_EXE} CI_EXEC_ROOT pip3 install --user $PIP_PACKAGES
```

Comment:
Is this needed?

Reply:
good catch, removed it and pip3 was still able to install.